assets/scripts/datalayer/common.js

```diff
@@ -112,7 +120,10 @@ document.shopAnalytics = {
   $(onLoad);
 
   $(document)
-    .on('click', '.products .product a', onProductClick);
+    .on('click', '.products .product a', onProductClick)
+    .on('click', document.shopAnalytics.event.click.login, onLoginFormSubmit)
+    .on('click', document.shopAnalytics.event.click.register, onRegisterFormSubmit)
+    .on('click', document.shopAnalytics.event.click.registerOnCheckoutSubmit, onRegisterOnCheckoutSubmit);
```

Comment:
The key in the settings is named `registerOnCheckout` now. Only the function name has the `Submit` suffix.

Reply:
Ack! That was a silly one 🙈 Sorry!